pkg/querier/querier.go

```diff
@@ -36,6 +40,10 @@ type Config struct {
 	metricsRegisterer prometheus.Registerer
 }
 
+var (
+	errBadLookbackConfigs = errors.New("bad settings, QueryStoreAfter >= QueryIngestersWithin which can result in queries not being sent")
```

Comment:
We should also replace `QueryStoreAfter` and `QueryIngestersWithin` with the user-configurable options here as well.

Reply:
done.